frappe/desk/reportview.py

```diff
@@ -150,13 +150,13 @@ def setup_group_by(data):
 			frappe.throw(_("Invalid aggregate function"))
 
 		if frappe.db.has_column(data.aggregate_on_doctype, data.aggregate_on_field):
-			data.fields.append(
-				"{aggregate_function}(`tab{aggregate_on_doctype}`.`{aggregate_on_field}`) AS _aggregate_column".format(
-					**data
-				)
-			)
+			column = f"`tab{data.aggregate_on_doctype}`.`{data.aggregate_on_field}`"
+			aggregate_function = data.aggregate_function
+
+			data.fields.append(f"{aggregate_function}({column}) AS _aggregate_column")
 			if data.aggregate_on_field:
-				data.fields.append(f"`tab{data.aggregate_on_doctype}`.`{data.aggregate_on_field}`")
+				data.fields.append(column)
+			data.group_by += f", {column}"
```

Comment:
This line seems to be causing an issue.

### Before

![Screenshot-2023-01-20-134639](https://user-images.githubusercontent.com/16315650/213648488-0b846994-37b0-4213-aedd-5898ee99b87f.png)


### After

![Screenshot-2023-01-20-134706](https://user-images.githubusercontent.com/16315650/213648499-4208d231-4f10-4408-b341-5fb80cb4d63b.png)